### PR TITLE
update event history status translation, remove unused translation

### PR DESCRIPTION
**Pull request by cibelius**
## Description

* Remove one unused translation
* Fix event audit history status translations to match v1
* Add translation for v2.verified

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/translations/client.csv [hunk 1]
```diff
@@ -2751,7 +2750,7 @@ v2.event.tennis-club-membership.title.label,Label for the summary title of the t
...
+v2.events.history.status,Events status history,"{status, select, CREATE {Draft} NOTIFY {Sent incomplete} VALIDATE {Validated} DRAFT {Draft} DECLARE {Sent for review} REGISTER {Registered} PRINT_CERTIFICATE {Certified} REJECT {Rejected} ARCHIVE {Archived} MARKED_AS_DUPLICATE {Marked as a duplicate} READ {Viewed} ASSIGN {Assigned} UNASSIGN {Unassigned} other {Unknown}}","{status, select, CREATE {Brouillon} NOTIFY {Envoyé incomplet} VALIDATE {Validé} DRAFT {Brouillon} DECLARE {Envoyé pour révision} REGISTER {Enregistré} PRINT_CERTIFICATE {Certifié} REJECT {Rejeté} ARCHIVE {Archivé} MARKED_AS_DUPLICATE {Marqué comme doublon} READ {Consulté} ASSIGN {Attribué} UNASSIGN {Désattribué} other {Inconnu}}"
```
Comment (jamil314) [inline src/translations/client.csv:2753]:
```suggestion
v2.events.history.status,Events status history,"{status, select, CREATE {Declaration started} NOTIFY {Sent incomplete} VALIDATE {Validated} DRAFT {Draft} DECLARE {Sent for review} REGISTER {Registered} PRINT_CERTIFICATE {Certified} REJECT {Rejected} ARCHIVE {Archived} MARKED_AS_DUPLICATE {Marked as a duplicate} READ {Viewed} ASSIGN {Assigned} UNASSIGN {Unassigned} other {Unknown}}","{status, select, CREATE {
Déclaration commencée} NOTIFY {Envoyé incomplet} VALIDATE {Validé} DRAFT {Brouillon} DECLARE {Envoyé pour révision} REGISTER {Enregistré} PRINT_CERTIFICATE {Certifié} REJECT {Rejeté} ARCHIVE {Archivé} MARKED_AS_DUPLICATE {Marqué comme doublon} READ {Consulté} ASSIGN {Attribué} UNASSIGN {Désattribué} other {Inconnu}}"
```

**Review by jamil314 — APPROVED** [review]
